src/__swaps__/screens/Swap/providers/SyncSwapStateAndSharedValues.tsx

```diff
@@ -74,16 +70,6 @@ export const SyncQuoteSharedValuesToState = () => {
   return null;
 };
 
-const getHasEnoughFundsForGas = (quote: Quote, gasFee: string, nativeNetworkAsset: ParsedAddressAsset | undefined) => {
```

Comment:
in this file, I'm basically removing anything for the `hasEnoughFundsForGas` so that we don't have to keep track of a separate state for this and instead incorporate Ben's maxSwappableGasWork to get to the same conclusion we care about (e.g, does the user have sufficient funds or not)

Reply:
does it still work if the input token is not the native one?

Reply:
whoops I'm an idiot - will fix